README.md

```diff
@@ -131,6 +131,8 @@ This module exposes a single function `download` which takes the same arguments
   * **parquet** loads the urls and optional caption as a parquet
 * **url_col** the name of the url column for parquet and csv (default *url*)
 * **caption_col** the name of the caption column for parquet and csv (default *None*)
+* **bbox_col** the name of the bounding box column (default *None*)
+* **bbox_format** the format of bounding boxes in the bounding box column. If set to *albumentations*, the bounding box has format ```[x_min, y_min, x_max, y_max]```. If set to *coco*, the bounding box has format ```[x_min, y_min, width, height]```. All are floats, relative to the height and width of the image. If *bbox_col* is not *None*, then this must also not be *None* (default *None*).
```

Comment:
is this still right ?

Reply:
do you think it's really useful to support both format ? I'm not sure if you saw my earlier comment

Reply:
My original thought was that this will be user defined, depending the dataset. See my comment below for details.